integrations/optimizely/lib/index.js

```diff
@@ -22,9 +22,7 @@ var Optimizely = (module.exports = integration('Optimizely')
   .option('variations', false) // send data via `.identify()`
   .option('listen', true) // send data via `.track()`
   .option('nonInteraction', false)
-  .option('sendRevenueOnlyForOrderCompleted', true)
-  .option('customExperimentProperties', {})
-  .option('customCampaignProperties', {}));
```

Comment:
Are there alternatives to Custom Experiment and Custom Campaign Properties with the new Optimizely Edge components?

Reply:
> Just noting that there are **no** instances where a customer has implemented these settings at the time of writing this.

Ah, I'd actually meant to ask about this! I'm not surprised, given what I was seeing in the code. Thank you so much for checking, and let us know if you need to introduce such a feature and don't know how.